AI_ML/CLASSIFICATION/TORCHSCRIPT_CLASSIFIER/TORCHSCRIPT_CLASSIFIER_test_.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+import os
+import tempfile
+import pandas as pd
+import numpy as np
+import PIL
+from flojoy import run_in_venv, Image, DataFrame
+
+
+@pytest.fixture
+def torchscript_model_path():
+    # Download and save a test model, this requires
+    # torch which is why we need to run this in a venv.
+    @run_in_venv(pip_dependencies=["torch~=2.0.1", "torchvision~=0.15.2"])
+    def _download_test_model(path: str):
+        import torch
+        import torchvision
+
+        class ModelWithTransform(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.model = torch.hub.load(
```

Comment:
[`torch.hub`'s models' cache depending on the presence of the `TORCH_HOME` environment variable](https://pytorch.org/docs/stable/hub.html#where-are-my-downloaded-models-saved).

Should we do another PR similar to https://github.com/flojoy-ai/python/pull/97 but to be able to set `TORCH_HOME` to Flojoy's cache directory?

Reply:
Good point